src/main/scala/com/kinja/amqp/MessageStore.scala

```diff
@@ -1,29 +1,44 @@
 package com.kinja.amqp
 
-import java.util.UUID
-
 import com.kinja.amqp.model.Message
 import com.kinja.amqp.model.MessageConfirmation
 
 trait MessageStore {
 
+	def saveConfirmation(conf: MessageConfirmation): Unit
+	def saveMessage(msg: Message): Unit
+
 	/**
-	 * Between start and commit, the message store should make sure that
+	 * Returns the number of deleted messages
+	 */
+	def deleteMessageUponConfirm(channelId: String, deliveryTag: Long): Int
```

Comment:
We should express the thing that it must return the number of deleted messages to work (like "Must return the number of deleted messages"). It could even return a `Boolean`, as the set of possible returned values is {0,1}